clang/docs/ReleaseNotes.rst

```diff
@@ -784,6 +784,7 @@ Miscellaneous Bug Fixes
 - Fixed an infinite recursion in ASTImporter, on return type declared inside
   body of C++11 lambda without trailing return (#GH68775).
 - Fixed declaration name source location of instantiated function definitions (GH71161).
+- Missing fallback to expression in the template differ when comparing ValueDecls.
```

Comment:
```suggestion
- Improve diagnostic output to print an expression instead of 'no argument` when comparing Values as template arguments.
```

Or something like this?